translations/base-pt-BR.yaml

```diff
@@ -0,0 +1,649 @@
+
+        scrollWheelSensitivity:
+            title: Sensibilidade do zoom
+            description: >-
+                Altera a sensibilidade do zoom (scrol do mouse ou touchpad).
+            sensitivity:
+                super_slow: Super lento
+                slow: Lento
+                regular: Normal
+                fast: Rápido
+                super_fast: Super Rápido
+                
+        fullscreen:
+            title: Tecla Cheia
+            description: >-
+                É recomendável jogar o jogo em tela cheia para obter a melhor experiência. Disponível apenas na versão completa.
+
+        soundsMuted:
+            title: Som
+            description: >-
+                Se ligado o jogo fica Mudo.
+
+        musicMuted:
+            title: Musica
+            description: >-
+                Se ligado a musica é desligada.
+
+        theme:
+            title: Tema
+            description: >-
+                Escolha o tema entre (Branco / Preto).
+
+        refreshRate:
+            title: Frequencia
+            description: >-
+                Se você possui um monitor de 144 hz, altere a taxa de atualização aqui para que o jogo seja simulado corretamente com taxas de atualização mais altas. Isso pode realmente diminuir o FPS se o computador estiver muito lento.
+
+        alwaysMultiplace:
+            title: Multiplicidade
+            description: >-
+                Se ativado, todos os edifícios permanecerão selecionados após o posicionamento até que você o cancele. Isso é equivalente a manter SHIFT permanentemente.
+
+        offerHints:
+            title: Dicas e tutoriais
+            description: >-
+                Se deve oferecer dicas e tutoriais enquanto estiver jogando.v.
+
+keybindings:
+    title: Comandos
+    hint: >-
+        Tip: Certifique-se de usar CTRL, SHIFT e ALT! Eles permitem diferentes opções de veiculação.
+
+    resetKeybindings: Reset Keyinbindings
+
+    categoryLabels:
+        general: Geral
+        ingame: Jogo
+        navigation: Navegação
+        placement: Posicionamento
+        massSelect: Seleção
+        buildings: Construções
+        placementModifiers: Modificações
+
+    mappings:
+        confirm: Confirmar
+        back: Voltar
+        mapMoveUp: Mover para cima
+        mapMoveRight: Mover para direita
+        mapMoveDown: Mover para baixo
+        mapMoveLeft: Mover para a esquerda
+        centerMap: Centralizar
+        
+        mapZoomIn: Aproximar
+        mapZoomOut: Distanciar
+        createMarker: Criar marcação
+
+        menuOpenShop: Atualizações
+        menuOpenStats: Estatísticas
+
+        toggleHud: Ocultar Menu
+        toggleFPSInfo: Mostar FPS
+        belt: *esteira
+        splitter: *divisor
+        underground_belt: *underground_belt
+        miner: *mineradora
+        cutter: *corte
+        rotater: *rotação
+        stacker: *empilhador
+        mixer: *misturador
+        painter: *pintor
+        trash: *lixo
```

Comment:
`&ptr` / `*ptr` in YAML as a pointer to another place, do not translate them
example:
```yaml
value: &myPointer 123
value2: *myPointer # will also be 123

```